main.tf

```diff
@@ -422,69 +422,90 @@ resource "zenml_service_connector" "gcp" {
   }
 }
 
 # Step Operator
 resource "zenml_stack_component" "step_operator" {
-  name      = "${var.zenml_stack_name == "" ? "terraform-vertex-${random_id.resource_name_suffix.hex}" : "${var.zenml_stack_name}-vertex"}"
-  type      = "step_operator"
-  flavor    = "vertex"
+  name   = var.zenml_stack_name == "" ? "terraform-vertex-${random_id.resource_name_suffix.hex}" : "${var.zenml_stack_name}-vertex"
+  type   = "step_operator"
+  flavor = "vertex"
 
   configuration = {
-    region = "${data.google_client_config.current.region}",
+    region          = "${data.google_client_config.current.region}",
     service_account = "${google_service_account.zenml_sa.email}"
   }
 
   connector_id = zenml_service_connector.gcp.id
 
   labels = {
-    "zenml:provider" = "gcp"
+    "zenml:provider"   = "gcp"
     "zenml:deployment" = "${var.zenml_stack_deployment}"
   }
 }
 
 # Image Builder
 resource "zenml_stack_component" "image_builder" {
-  name      = "${var.zenml_stack_name == "" ? "terraform-gcp-${random_id.resource_name_suffix.hex}" : "${var.zenml_stack_name}-gcp"}"
-  type      = "image_builder"
-  flavor    = "gcp"
+  name   = var.zenml_stack_name == "" ? "terraform-gcp-${random_id.resource_name_suffix.hex}" : "${var.zenml_stack_name}-gcp"
+  type   = "image_builder"
+  flavor = "gcp"
 
   configuration = {}
 
   connector_id = zenml_service_connector.gcp.id
 
   labels = {
-    "zenml:provider" = "gcp"
+    "zenml:provider"   = "gcp"
+    "zenml:deployment" = "${var.zenml_stack_deployment}"
+  }
+}
+
+# Experiment Tracker
+resource "zenml_stack_component" "experiment_tracker" {
+  name   = var.zenml_stack_name == "" ? "terraform-gcp-${random_id.resource_name_suffix.hex}" : "${var.zenml_stack_name}-gcp"
+  type   = "experiment_tracker"
+  flavor = "vertex"
+
+  configuration = {
+    project        = "${data.google_client_config.current.project}",
+    location       = "${data.google_client_config.current.region}"
+    staging_bucket = "gs://${google_storage_bucket.artifact_store.name}"
+  }
+
+  connector_id = zenml_service_connector.gcs.id
+
+  labels = {
+    "zenml:provider"   = "gcp"
     "zenml:deployment" = "${var.zenml_stack_deployment}"
   }
 }
 
 # Complete Stack
 resource "zenml_stack" "stack" {
-  name = "${var.zenml_stack_name == "" ? "terraform-gcp-${random_id.resource_name_suffix.hex}" : var.zenml_stack_name}"
+  name = var.zenml_stack_name == "" ? "terraform-gcp-${random_id.resource_name_suffix.hex}" : var.zenml_stack_name
 
   components = {
     artifact_store     = zenml_stack_component.artifact_store.id
     container_registry = zenml_stack_component.container_registry.id
-    orchestrator      = zenml_stack_component.orchestrator.id
+    orchestrator       = zenml_stack_component.orchestrator.id
     step_operator      = zenml_stack_component.step_operator.id
     image_builder      = zenml_stack_component.image_builder.id
+    experiment_tracker = zenml_stack_component.experiment_tracker.id
```

Comment:
Could you please also update the README file to mention this new stack component ?